.github/workflows/test-deploy.yml

```diff
@@ -39,7 +39,7 @@ jobs:
 
   test_remote:
     name: Superset remote deployment
-    if: github.base_ref == 'main'
```

Comment:
Why change this?

Reply:
test-deploy for remote deployment otherwise was being skipped, temporary

Reply:
reverted back to main